docs/website/docs/dlt-ecosystem/verified-sources/rest_api/basic.md

```diff
@@ -685,22 +702,105 @@ def repositories() -> Generator[Dict[str, Any]]:
     yield from [{"name": "dlt"}, {"name": "verified-sources"}, {"name": "dlthub-education"}]
 ```
 
+### Processing steps: filter and transform data
 
-#### Include fields from the parent resource
+The `processing_steps` field in the resource configuration allows you to apply transformations to the data fetched from the API before it is loaded into your destination. This is useful when you need to filter out certain records, modify the data structure, or anonymize sensitive information.
 
-You can include data from the parent resource in the child resource by using the `include_from_parent` field in the resource configuration. For example:
+Each processing step is a dictionary specifying the type of operation (`filter` or `map`) and the function to apply.
```

Comment:
```suggestion
Each processing step is a dictionary specifying the type of operation (`filter` or `map`) and the function to apply. Steps apply in the order they are listed.
```

Reply:
Idea: What do you think about mentioning it here already?
It's certainly clear due to the tip box given below! Not sure how much redundancy we want.

Reply:
I think it won't hurt :)